pathfinder/api/rest.py

```diff
@@ -147,7 +147,7 @@ def put(self, token_network_address: str, channel_id: str):
         if channel_id_error is not None:
             return channel_id_error
 
-        channel_id = int(channel_id)
+        channel_id_casted = int(channel_id)
```

Comment:
you can use flask's request parser to validate the argument. And if I remember correctly, you can also force int type for part of the url.

https://github.com/raiden-network/microraiden/blob/master/microraiden/proxy/paywalled_proxy.py#L76

Reply:
Good point, but we decided to do that later and introduce our own converters then.